README.rst

```diff
@@ -80,24 +80,25 @@ been used to create the diagram.
 
 Installation
 ------------
-We highly recommend to use anaconda_ and install straditize::
+We highly recommend to use anaconda_ and install straditize into an own environment::
 
     conda config --add channels conda-forge
-    conda install -c chilipp straditize
+    conda create -n straditize -c chilipp straditize
+    conda activate straditize
 
 or install it from the source files via::
 
     git clone https://github.com/Chilipp/straditize.git
     cd straditize
-    python setup.py install
+    pip install .  # or python setup.py install, but pip is recommended
 
 It can then be started from the command line via::
 
     straditize
 
 A more detailed description is provided in the docs_.
 
-.. _anaconda: https://www.continuum.io/downloads
+    .. _anaconda: https://conda.io/en/latest/miniconda.html
```

Comment:
Not sure if the indentation here matters

Reply:
Thanks! This is resolved by https://github.com/Chilipp/straditize/pull/6/commits/5522234bca9dbad38072e85769addc84145a7096